usecases/config/environment.go

```diff
@@ -225,6 +225,10 @@ func FromEnv(config *Config) error {
 	if v := os.Getenv("AUTOSCHEMA_DEFAULT_DATE"); v != "" {
 		config.AutoSchema.DefaultDate = v
 	}
+	config.AutoSchema.AutoTenantsEnabled = true
+	if v := os.Getenv("AUTOSCHEMA_AUTOTENANTS_ENABLED"); v != "" {
+		config.AutoSchema.AutoTenantsEnabled = !(strings.ToLower(v) == "false")
+	}
```

Comment:
hmm should we have a per-class setting as part of the multi-tenancy config? That would also make testing much easier

Reply:
I like the idea. Can you elaborate on how this might look?

Reply:
```
		paragraphClass.MultiTenancyConfig = &models.MultiTenancyConfig{
			Enabled: true, AutoTenantCreation: true/false
		}
```

how about this?